src/eventer/eventer_epoll_impl.c

```diff
@@ -177,9 +179,14 @@ static eventer_t eventer_epoll_impl_remove(eventer_t e) {
       removed = e;
       master_fds[e->fd].e = NULL;
       if(epoll_ctl(spec->epoll_fd, EPOLL_CTL_DEL, e->fd, &_ev) != 0) {
-        mtevL(mtev_error, "epoll_ctl(%d, EPOLL_CTL_DEL, %d) -> %s\n",
-              spec->epoll_fd, e->fd, strerror(errno));
-        if(errno != ENOENT) abort();
+        if(errno != ENOENT) {
+          mtevFatal(mtev_error, "epoll_ctl(%d, EPOLL_CTL_DEL, %d) -> %s\n",
+                spec->epoll_fd, e->fd, strerror(errno));
+        }
+        else {
+          mtevL(mtev_error, "epoll_ctl(%d, EPOLL_CTL_DEL, %d) -> %s\n",
+                spec->epoll_fd, e->fd, strerror(errno));
+        }
```

Comment:
Stylistically, the Cut-n-Paste log message bothers me... Not a big deal, but what if you left the old `mtevL()` call, then had `mtevFatal(mtev_error, "errno != ENOENT: %d (%s)\n", errno, strerror(errno));` in the abort case?